libs/cardano-ledger-pretty/cardano-ledger-pretty.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:      3.0
 name:               cardano-ledger-pretty
-version:            1.3.0.1
```

Comment:
@lehins  - I had to make these changes to because of the conway version change

Reply:
That's right, it was expected.

Reply:
Thank you for doing the change!